web/src/user_topics_ui.ts

```diff
@@ -76,7 +76,7 @@ export function handle_topic_updates(user_topic_event: ServerUserTopic): void {
     setTimeout(() => {
         // Defer updates for any background-rendered messages lists until the visible one has been updated.
         for (const list of message_lists.all_rendered_message_lists()) {
-            if (list.preserve_rendered_state && message_lists.current !== list) {
+            if (message_lists.current !== list) {
                 list.update_muting_and_rerender();
```

Comment:
message_list: Remove unused preserve_rendered_state attribute.

If this property was never set, then this condition was always false, right? So removing the unused property would mean deleting this code.

Reply:
Yeah, deleting would be the right way to conserve the current behavior. I wonder, though, if `list.preserve_rendered_state` should have been migrated to `list.should_preserve_current_rendered_state()` by #30062 (cc @amanagr)?

Reply:
> but @amanagr said [above](https://github.com/zulip/zulip/pull/31591#pullrequestreview-2300886181) that we should be running this even if it wasn’t running before.

I see, OK. If that's what we want, then the commit message should just reflect that that's the intention.

Reply:
Oh oops sorry, I thought I'd put that in the commit message already.

Reply:
I think `list.should_preserve_current_rendered_state()` determines whether this is present in `all_rendered_message_lists`, so I think the check can be just deleted.